data-share/data-share-service/src/main/java/io/mosip/datashare/exception/StaticDataShareException.java

```diff
@@ -0,0 +1,29 @@
+package io.mosip.datashare.exception;
+
+import io.mosip.datashare.constant.DataUtilityErrorCodes;
+import io.mosip.kernel.core.exception.BaseUncheckedException;
+
```

Comment:
Please provide the author for new class

Reply:
Added the author to StaticDataShareException.java class.